tutorenterprisecatalog/templates/enterprisecatalog/tasks/lms/init

```diff
@@ -0,0 +1,90 @@
+# Modify users created with an incorrect email and that might clash with the newly created users
+./manage.py lms shell -c \
+    "from django.contrib.auth import get_user_model;\
+    get_user_model().objects.filter(username='enterprise-catalog').exclude(email='enterprise-catalog@openedx').update(email='enterprise-catalog@openedx')"
+./manage.py lms shell -c \
+    "from django.contrib.auth import get_user_model;\
+    get_user_model().objects.filter(username='enterprise_worker').exclude(email='enterprise_worker@openedx').update(email='enterprise_worker@openedx')"
+./manage.py lms shell -c \
+    "from django.contrib.auth import get_user_model;\
+    get_user_model().objects.filter(username='enterprise-catalog-worker').exclude(email='enterprise-catalog-worker@openedx').update(email='enterprise-catalog-worker@openedx')"
+
+./manage.py lms manage_user enterprise-catalog enterprise-catalog@openedx --staff --superuser --unusable-password
+./manage.py lms manage_user enterprise-catalog-worker enterprise-catalog-worker@openedx --staff --superuser --unusable-password
+
+# Enterprise backend user and permissions
+./manage.py lms manage_user enterprise_worker enterprise_worker@openedx --staff --superuser --unusable-password
+./manage.py lms shell -c \
+    "from django.contrib.auth import get_user_model;\
+    from django.contrib.auth.models import Permission;\
+    enterprise_worker = get_user_model().objects.get(username='enterprise_worker');\
+    enterprise_model_permissions = list(Permission.objects.filter(content_type__app_label='enterprise'));\
+    enterprise_worker.user_permissions.add(*enterprise_model_permissions);\
+    enterprise_worker.save()
```

Comment:
Why do we need this? I didn't notice these permissions in the existing approach.

Reply:
It is part of [provision script](https://github.com/openedx-unsupported/devstack/blob/323b475b885a2704489566b262e2895a4dca62b6/enterprise/provision.sh#L6) in devstack.